src/material/select/select.e2e.spec.ts

```diff
@@ -11,8 +11,8 @@ describe('select', () => {
   // accidentally. This could happen because the select panel is removed from DOM
   // immediately when an option is clicked. Usually ripples still fade-in at that point.
   it('should not accidentally persist ripples', async () => {
-    const select = getElement('.mat-select');
-    const options = element.all(by.css('.mat-option'));
+    const select = getElement('.mat-mdc-select');
+    const options = element.all(by.css('.mat-mdc-option'));
     const ripples = element.all(by.css('.mat-ripple-element'));
```

Comment:
Now this test doesn't run at all. Is that intentional or do I miss something? Looks like the file needs to be moved to the non-legacy build package

Reply:
This is the non-legacy package. The legacy select is in `material/legacy-select` now.

Reply:
oh I was thinking it was part of the legacy BUILD package. sounds good